src/metatrain/experimental/pet/utils/systems_to_batch_dict.py

```diff
@@ -5,6 +5,21 @@
 from metatensor.torch.atomistic import NeighborListOptions, System
 
 
+def process_neighbors(
```

Comment:
why do you redefine a function that just calls the operation?

Reply:
Not sure 100%. I took this piece of code from the original pet repo, and there was a reason (maybe TorchScript related) to wrap this up. I'll try to remove the wrapper and we'll see how it goes

Reply:
the only reason I could see would be to use keyword arguments, but there is a way to do this directly with the C++ operation (requiring some small changes to the function registration)